clouddriver-web/src/main/groovy/com/netflix/spinnaker/clouddriver/controllers/SubnetController.groovy

```diff
@@ -31,10 +32,27 @@ class SubnetController {
   List<SubnetProvider> subnetProviders
 
   @RequestMapping(method = RequestMethod.GET)
-  Set<Subnet> list() {
-    subnetProviders.collectMany {
+  Map<String, Set<Subnet>> list() {
```

Comment:
I found this style (i.e. /{cloudProvider}) in use by CacheController, ClusterController, NetworkController and OperationsController.
The cloudProvider attribute can be used for filtering if necessary by the caller, but would be a breaking change to rely on the caller to do that anyway.
Chose to explicitly break (and fix) gate as the only caller of clouddriver, and not break any callers of gate.
This overall pattern matches that of NetworkController.